pkg/api/request.go

```diff
@@ -36,14 +36,20 @@ func (a *API) UserListRequestsUpcoming(w http.ResponseWriter, r *http.Request, p
 		RequestEndComparator: storage.GreaterThan,
 		CompareTo:            time.Now(),
 	}
-	_, err := a.DB.Query(ctx, &q, queryOpts...)
+	qR, err := a.DB.Query(ctx, &q, queryOpts...)
```

Comment:
Real nitpick here 😆 can this just be qr instead of qR
```suggestion
	qr, err := a.DB.Query(ctx, &q, queryOpts...)
```